prov/verbs/src/verbs_mr.c

```diff
@@ -357,11 +357,11 @@ vrb_mr_reg_iface(struct fid *fid, const void *buf, size_t len, uint64_t access,
 			      util_domain.domain_fid.fid);
 
 	if (domain->cache.monitors[iface])
-		return vrb_mr_cache_reg(domain, buf, len, access, offset,
+		return vrb_mr_cache_reg(domain, buf, len, access, NULL, offset,
```

Comment:
The verbs MR cache path doesn't support DMA buf right now. I did not add that support.

Reply:
We shouldn't try to cache dmabuf registrations.  The cache is mainly for internal registrations.

Reply:
Why is that?

Reply:
Open MPI 5.0 is doing fi_mr_regattr for each fi_tsenddata https://github.com/open-mpi/ompi/blob/fee1d707432d4586cf5d1df8f3e3fd2aeff52469/ompi/mca/mtl/ofi/mtl_ofi.h#L340. It relies on the MR cache to reduce the costs of MR registration for both host and hmem memory.